packages/rich-text-editor/src/RichTextEditor/schema.ts

```diff
@@ -47,5 +48,19 @@ export const createSchemaFromControls = controls => {
     }),
     {}
   )
-  return new Schema({ nodes, marks: newMarks })
+
+  // TODO: This is hard coded to always enable lists for now,
+  // but once we have the toolbar we can use the controls to determine this.
+  const listsEnabled = true
+
+  const schema = new Schema({ nodes, marks: newMarks })
+
+  if (listsEnabled) {
+    return new Schema({
+      nodes: addListNodes(schema.spec.nodes, "paragraph block*", "block"),
```

Comment:
Bit of a gotcha here, this `addListNodes` function has to be run on a schema that's built, rather than just the nodes object. 

https://discuss.prosemirror.net/t/nodes-append-is-not-a-function/2216